tools/linter/yamllint/.yamllint

```diff
@@ -37,7 +37,7 @@ rules:
     max-spaces-after: 1
   indentation:
     spaces: 2
-    indent-sequences: whatever
+    indent-sequences: consistent
```

Comment:
nit: can you add a comment why this was needed ?

Reply:
It was desired rather than needed, FTR. 🙂 Done.